docs/authzed/pricing.md

```diff
@@ -0,0 +1,93 @@
+# Pricing on Authzed
+
+[Authzed], the managed permissions database service, has a **usage** based pricing model.
+
+Customers are charged based on two dimensions: the number of [relationships] stored in Authzed, and the number of dispatched operations made when performing [API] calls.
+
+[relationships]: /reference/glossary#relationship
+[API]: /reference/api
+[Authzed]: https://app.authzed.com
+
+| Dimension | Price Per 1 miliion |
+| --------- | ------------------- |
+| Relationships | $1 per 1 million |
+| Dispatched Operations | $1 per 1 million |
+
+## Do you have a free tier?
+
+Yes. Any amount less than 1 million relationships or dispatched operations is considered a charge of `$0`.
+
+## What counts as a "dispatched operation"?
+
+A **dispatched operation** is a single *non-cached* operation performed by Authzed to resolve the answer to the API request.
+
+The number of dispatched operations depends heavily on the type of [API] being called, the complexity of the permissions system's [schema] and how many of those operations have been cached from previous [API] calls.
+
+[schema]: /guides/schema
+
+### API Dispatch Calculation
```

Comment:
Calculation => Estimation since this is not a perfect way to determine the charge.